src/index.ts

```diff
@@ -830,11 +831,25 @@ export type PreparedFn<T extends Dictionary<ParameterAlias>, U> = (
 	passthrough?: Params['passthrough'],
 ) => U;
 
+export type RetryParameters = {
+	canRetry?: (err: any) => boolean;
+	logger?: (
+		prevErr: any,
+		delayMs: number,
+		attempt: number,
+		isLastAttempt: boolean,
```

Comment:
I'd prefer to make this
```suggestion
		maxAttempts: number,
```
as I think it's overall more useful (and you should be able to do `isLastAttempt = attempt === maxAttempts` if you still want that specific info)

Note: this is coupled with other comments where I think `maxAttempts` should be required and separate from the max delay